server/src/main/java/org/elasticsearch/index/translog/Checkpoint.java

```diff
@@ -196,7 +196,7 @@ public static void write(ChannelFactory factory, Path checkpointFile, Checkpoint
         };
         final String resourceDesc = "checkpoint(path=\"" + checkpointFile + "\", gen=" + checkpoint + ")";
         try (OutputStreamIndexOutput indexOutput =
-                 new OutputStreamIndexOutput(resourceDesc, checkpointFile.toString(), byteOutputStream, V3_FILE_SIZE)) {
+                 new OutputStreamIndexOutput(resourceDesc, checkpointFile.toString(), byteOutputStream, V3_FILE_SIZE, true)) {
```

Comment:
That true turns on encryption (as Checkpoint already attempts to decrypt via SimpleFSDirectory)